lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -25,7 +25,7 @@ def as_json
           meta = meta_for(association)
           hash[:meta] = meta if meta
```

Comment:
I don't want to have too many placeholders right now. Overall, we're trying to simplify AMS. :-)